Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+@Library('xmos_jenkins_shared_library@v0.16.2') _
+
+getApproval()
+
+pipeline {
+  agent {
+    label 'x86_64 && brew && macOS'
+  }
+  environment {
+    REPO = 'test_support'
+    VIEW = getViewName(REPO)
+  }
+  options {
+    skipDefaultCheckout()
+  }
+  stages {
+    stage('Checkout') {
+      steps {
+        script {
+          def current_scm = checkout scm
+          env.SAVED_GIT_URL = current_scm.GIT_URL
+          env.SAVED_GIT_COMMIT = current_scm.GIT_COMMIT
+        }
+      }
+    }
```

Comment:
This should probably read:
```
stage('Get view') {
  steps {
    xcorePrepareSandbox("${VIEW}", "${REPO}")
  }
}
```
You will also need a new viewfile in xgit called "test_support_develop.txt" that includes infr_scripts_pl (to get the build scripts), tools_released ( to get the tools like xsim), infr_apps (because thats where the library checking scripts are) and infr_scripts_py (since that appears in the requirements.txt).